cdap-unit-test/src/main/java/co/cask/cdap/test/internal/DefaultStreamManager.java

```diff
@@ -147,6 +157,36 @@ public void send(Map<String, String> headers, ByteBuffer buffer) throws IOExcept
   }
 
   @Override
+  public void send(File file, String contentType) throws Exception {
+    String path = String.format("/v3/namespaces/%s/streams/%s/batch", streamId.getNamespaceId(), streamId.getId());
+    HttpRequest request = new DefaultHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.POST, path);
+    byte[] fileContent = Files.readAllBytes(Paths.get(file.toURI()));
```

Comment:
my bad. I had tried two approaches, this was left over from the approach that didn't work. removed.